docs/releases/1.9.txt

```diff
@@ -341,6 +341,10 @@ Generic Views
 * :func:`~django.utils.decorators.method_decorator` can now be used to
   :ref:`decorate classes instead of methods <decorating-class-based-views>`.
 
+* :func:`~django.utils.decorators.method_decorator` can now be used with
```

Comment:
combine with previous bullet. "It can now also be passed an iterable of decorators."